lib/osut/utils.rb

```diff
@@ -1714,94 +1837,224 @@ def airLoopsHVAC?(model = nil)
+
+      if cnd.empty?
+        cnd = nil
+      else
+        cnd = cnd.get.downcase
 
-    id  = space.nameString
-    ok1 = [true, false].include?(loops)
-    ok2 = [true, false].include?(setpoints)
-    return mismatch(id, space, cl, mth, DBG, false) unless space.is_a?(cl)
-    return invalid("loops",     mth, 2, DBG, false) unless ok1
-    return invalid("setpoints", mth, 3, DBG, false) unless ok2
+        if cts.include?(cnd)
+          return res if cnd == "unconditioned"
+        else
+          cnd = nil
+          invalid("#{tag}:#{cnd}", mth, 0, ERR)
+        end
+      end
+    end
 
-    return false if plenum?(space, loops, setpoints)
-    return false if space.thermalZone.empty?
-    return true  if space.thermalZone.get.thermostat.empty?
+    # 2. Check instead OSut's INDIRECTLYCONDITIONED (parent space) link.
+    if cnd.nil?
+      id = space.additionalProperties.getFeatureAsString(tg2)
 
-    false
+      unless id.empty?
+        id  = id.get
+        dad = space.model.getSpaceByName(id)
+
+        if dad.empty?
+          log(ERR, "Unknown space #{id} (#{mth})")
+        else
+          # Now focus on 'parent' space linked to INDIRECTLYCONDITIONED space.
+          space = dad.get
+          cnd   = tg2
+        end
+      end
+    end
+
+    # 3. Fetch space setpoints (if model indeed holds valid setpoints).
+    heated = heatingTemperatureSetpoints?(space.model)
+    cooled = coolingTemperatureSetpoints?(space.model)
+    zone   = space.thermalZone
+
+    if heated || cooled
+      return res if zone.empty? # UNCONDITIONED
+
+      zone = zone.get
+      res[:heating] = maxHeatScheduledSetpoint(zone)[:spt]
+      res[:cooling] = minCoolScheduledSetpoint(zone)[:spt]
+    end
+
+    # 4. Reset if AdditionalProperties were found & valid.
+    unless cnd.nil?
+      if cnd == "unconditioned"
+        res[:heating] = nil
+        res[:cooling] = nil
+      elsif cnd == "semiheated"
+        res[:heating] = 15.0 if res[:heating].nil?
+        res[:cooling] = nil
+      elsif cnd.include?("conditioned")
+        # "nonresconditioned", "resconditioned" or "indirectlyconditioned"
+        res[:heating] = 21.0 if res[:heating].nil? # default
+        res[:cooling] = 24.0 if res[:cooling].nil? # default
+      end
+    end
+
+    # 5. Reset if plenum?
+    if plenum?(space)
+      res[:heating] = 21.0 if res[:heating].nil? # default
+      res[:cooling] = 24.0 if res[:cooling].nil? # default
+    end
+    
+    res
   end
 
+  ##
+  # Validates if a space is UNCONDITIONED.
+  #
+  # @param space [OpenStudio::Model::Space] a space
+  #
+  # @return [Bool] true if space can be considered as UNCONDITIONED
+  # @retrun [Bool] false if invalid input (see logs)
+  def unconditioned?(space = nil)
+    mth = "OSut::#{__callee__}"
+    cl  = OpenStudio::Model::Space
+    return mismatch("space", space, cl, mth, DBG, false) unless space.is_a?(cl)
+
+    ok = false
+    ok = setpoints(space)[:heating].nil? && setpoints(space)[:cooling].nil?
```

Comment:
An _indirectly-conditioned_ space (e.g. return air plenum) will return the heating and/or cooling setpoints of its linked space (e.g. below), if so enabled. Otherwise, a space simply tagged as a plenum will return defaulted heated/cooled setpoints (21°C/24°C) as a proxy. So _unconditioned?_ returns FALSE for plenums and other indirectly-conditioned spaces.